packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+import { apiURL } from '../../apiURL';
+import { Typography, Grid } from '@mui/material';
+import MovieCard from '../MovieCard/MovieCard';
+import MovieCard2 from '../MovieCard/MovieCard2';
+
+export default function LatestRatingMovies() {
+  const [movies, setMovies] = useState([]);
+  useEffect(() => {
+    async function fetchMovies() {
+      const response = await fetch(`${apiURL()}/movies`);
+      const examples = await response.json();
+      setMovies(examples);
+    }
+    fetchMovies();
+  }, []);
```

Comment:
this is the happy scenario, perhaps you want to add loading and error scenario as well. Doesn't have to be a part of this PR :)

Reply:
Hi 
Thanks for review,I made this with dummy data as back end api still is not working :).

Reply:
Ok, I will add it :)